src/components/DocLink.tsx

```diff
@@ -21,61 +15,37 @@ export type DocLinkProps = {
 }
 
 const DocLink = ({ href, children, isExternal = false }: DocLinkProps) => {
-  const linkBoxShadowColor = useToken("colors", "primary.base")
-  const { flipForRtl } = useRtlFlip()
+  const { isRtl } = useRtlFlip()
 
   return (
     <LinkBox
-      padding={4}
-      borderRadius="sm"
-      color="text"
-      border="1px"
-      borderStyle="solid"
-      borderColor="border"
-      _hover={{
-        background: "tableBackgroundHover",
-        borderRadius: "base",
-        boxShadow: `0 0 1px ${linkBoxShadowColor}`,
-      }}
+      className={cn(
+        "flex rounded-sm border p-4 text-current no-underline",
+        "hover:rounded hover:bg-background-highlight hover:no-underline",
+        "group hover:shadow-[0_0_1px] hover:shadow-primary"
+      )}
     >
-      <Flex
-        zIndex={1}
-        display="flex"
-        flexDirection="row"
-        flex={1}
-        justifyContent="space-between"
-        data-group
-      >
-        <Flex align="center">
+      <Flex className="flex-1 justify-between">
+        <Center>
           <Emoji className="me-4 text-md" text=":page_with_curl:" />
-        </Flex>
-        <Box flex={1} flexDirection="column">
-          <LinkOverlay
-            href={href}
-            as={BaseLink}
-            isExternal={isExternal}
-            textDecoration="none"
-            _hover={{ textDecoration: "none" }}
-            hideArrow
-          >
-            <Text color="text300" fontWeight="semibold" margin={0}>
+        </Center>
+        <Stack className="flex-1">
+          <LinkOverlay href={href} className="no-underline">
+            {" "}
+            <p className="font-semibold text-[#4c4c4c] dark:text-[#cccccc]">
```

Comment:
We don't actually have `semibold` (font-weight 600) installed, so this will end up just using bold. 


Would advocate for at least using colors from the gray pallet here, instead of one-off hex values:

```suggestion
            <p className="font-bold text-gray-600 dark:text-gray-200">
```

Reply:
<img width="827" alt="image" src="https://github.com/user-attachments/assets/ea0f79fd-1b28-4a0a-bdcd-6242b98c266f">
<img width="920" alt="image" src="https://github.com/user-attachments/assets/41d5fccc-010a-4606-82d5-d88abd8e0cc7">

Reply:
On it ser 🫡